### ISSUE TITLE
[Adaptive Cards - Stock Update]: Talkback does not announce the information about the stocks going up on Stock Update pane.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#102]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

Repro Steps:
1. Launch the Adaptive cards app on Android.
2.  Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Stock Update'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.
8. Turn On talkback feature and observe the issue.

**Actual Result:**
Talkback does not announces the information about the stocks going up on Stock Update pane. Talkback is announcing, "2.69 USD 2.13%".

**Expected Result:**
Talkback should announce the complete information about the content present on Stock Update pane as, "2.69 USD Stocks are rising by 2.13%" or "Stock increment by 2.13%".

**User Experience:**
Users who depends on AI might get confused if Talkback is not announcing the complete information about the content present on Stock Update pane.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/83b3f60b-c378-49fb-8954-ac5011b8d745

![Talkback does not announce the information about the stocks going up on Stock Update pane](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/e750b203-015e-4ecf-89b4-c5ac72aed114)

### Card JSON

```JSON
NA
```
